### ISSUE TITLE
Which image file formats to choose?

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/bioimage-analysis-training-standards#5]
I have a CZI file that I would like to contribute.

Should I convert it? To what?

My personal opinion: 

It is useful to also have commercial file formats, but _only_ for teaching how to deal with those file formats.

Data that is of general use should be in a wide spread and easy to deal with community standard file format. 
I think currently there the only option is OME.TIFF?!

What do you think? ping @haesleinhuepf @RoccoDAnt @ArneS70 @biovoxxel

**Comment by biovoxxel** [comment]
I think all tools we normally use for teaching run Bio-Formats and can open those readable "standard" formats. Therefore, I think if it is not the strangest format ever, I would keep the original format (also for the sake of not messing up with the metadata. 
For very problematic ones, we can think about a conversion. But I am the wrong person to opinionate on which is the best format.

**Comment by ArneS70** [comment]
I agree with @biovoxxel . 
I also recommend to not convert image data as long as it is readable with BioFormats unless there are good reasons (e.g. pyramidal data format needed).

**Comment by tischi** [comment]
OK, I wanted to disagree and say we should support community standards, but experiments that I did today appear to have [such a FRUSTRATING outcome](https://forum.image.sc/t/bio-formats-color-wrong-for-imagej-images/76021/5) that I indeed start to wonder whether, e.g., CZI is preferable over OME.TIFF. 

I am probably overreacting right now, but I spent the last couple of days dealing with various image file format issues and I am sort of loosing the patience a little bit ;-)

**Comment by haesleinhuepf** [comment]
I recently wrote [this chapter out of similar frustration](https://haesleinhuepf.github.io/BioImageAnalysisNotebooks/12a_image_file_formats/readme.html). In Python it's much worse compared to Java/ImageJ btw.

And I agree with the above comments. Let's keep files in the format they come if possible.

**Comment by tischi** [comment]
Yeah, I thought that at least ImageJ and Bio-Formats would play well along together. To learn that they don't is a bit of a shocker I must say....
